pytplot/tplot_math/interpolate.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Regents of the University of Colorado. All Rights Reserved.
+# Released under the MIT license.
+# This software was developed at the University of Colorado's Laboratory for Atmospheric and Space Physics.
+# Verify current version before use at: https://github.com/MAVENSDC/Pytplot
+
+import pytplot
+import pydivide
+import numpy as np
+from scipy import interpolate
+from scipy.interpolate import interp1d
+import pandas as pd
+
+#TVAR INTERPOLATION
+#interpolate tvar2 to tvar1 cadence
+def interpolate(tvar1,tvar2,interp='linear'):
+    #crop data
+    tv1_t,tv1_d,tv2_t,tv2_d = pytplot.crop(tvar1,tvar2)
+    df_index = pytplot.data_quants[tvar1].data.columns
```

Comment:
I think this should be for tvar2?  Because later you loop through tvar2's columns with this info